src/System.Private.DataContractSerialization/src/System/Xml/XmlUTF8TextWriter.cs

```diff
@@ -5,6 +5,7 @@
 using System.Runtime;
```

Comment:
Sorry - disregard my previous message, I thought this was copy of one of System.Xml.ReaderWriter files.